tileserver/__init__.py

```diff
@@ -38,12 +40,26 @@ def coord_is_valid(coord):
     return True
 
 
-RequestData = namedtuple('RequestData', 'layer_spec coord format')
+RequestData = namedtuple('RequestData', 'layer_spec coord format params')
```

Comment:
I wonder if `params` could be more specific. It looks like it might be `style_params` below?